### PR TITLE
Added a webapp to see likes, comments and other controls. And callbacks

### PR DESCRIPTION
**Pull request by subins2000**
I've made a small webapp called 'instalv' to see the likes and comments while the stream is going on.

For this, I've modified the `goLive.php` script to `live.php` which listens for commands in `request` file and outputs the response to a `response` file.

The webapp is ran with PHP Built-in web server which writes command to this `request` file and then display the response from `response`.

I also added the capability in `live.php` for callbacks when a like or comment occurs. For personal use, I've used the callbacks to display the like and comment as a notification

screenshot from 2018-07-31 21-04-38

### CODE REVIEW

**Comment by JRoy** [comment]
Holy Moly, there are a lot of commits here! I'll be looking over this for a few days to evaluate how this looks.

Thanks for your contrubution!

File: instalv/live.php [hunk 1]
```diff
@@ -0,0 +1,269 @@
+<?php
```
Comment (JRoy) [inline instalv/live.php]: This file looks like a lot of repeat from the orignal live script, could you please look into merging the two and keeping all files in the same directory. (Just for simplcity)

File: instalv/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* Change the config inside `config.php`
+* Open a terminal and run the `live.php` :
+    ```bash
+    php live.php
+    ```
+* Open another terminal and start a local server
+    ```bash
+    php -S localhost:8000 server.php
+    ```
+    For easy use, there's a `run_server.sh` script which will do this.
```
Comment (JRoy) [inline instalv/README.md]: Also, a batch script could be useful for windows users as well

File: instalv/config.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+define('IG_USERNAME', 'USERNAME');
+define('IG_PASS', 'PASSWORD');
+
+$cfg_callbacks = [
```
Comment (JRoy) [inline instalv/config.php]: If it's not a pain, maintaining windows compatibility would be nice

File: instalv/live.php [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+    logM("Something Went Super Wrong! Attempting to At-Least Clean Up!");
+
+    $ig->live->getFinalViewerList($broadcastId);
+    $ig->live->end($broadcastId);
+} catch (\Exception $e) {
+    echo 'Error While Creating Livestream: '.$e->getMessage()."\n";
+}
+
+function addLike($user) {
```
Comment (JRoy) [inline instalv/live.php]: If you could document this code so people can better understand what it's doing

File: instalv/server.php [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                        <label>seconds.<label>
+                    </div>
+                    <div>
+                        Pause : <input type="checkbox" id="pause_refresh" />
+                    </div>
+                </div>
+                <div class="col-8">
+                    <h2>Commands</h2>
+                    <button type="button"class="btn btn-danger" id="end_stream">End Stream</button>
+                    <button type="button"class="btn btn-danger" id="get_stream_info" data-cmd="stream_info">Stream Key/URL</button>
```
Comment (JRoy) [inline instalv/server.php]: It dosn't have to but if these added 2 new fields for the key and url seperatly, that would be nice

**Comment by JRoy** [comment]
@subins2000 have you seen the requested changes for merge?

**Comment by subins2000** [comment]
@JRoy Yes I have. Wasn't able to get the time to review and fix it

**Comment by JRoy** [comment]
@subins2000 Ah okay, just ping me once you do fix it and I'll re-review it!

**Comment by penteseter123** [comment]
Added a webapp to see likes, comments and other controls. And callbacks

I want Instal InstaLV in Windows 10 why ?

**Comment by subins2000** [comment]
I've decided to [fork this project away](https://github.com/subins2000/InstaLV). Mainly cause the `live.php` of mine and `goLive.php` of this repo differs a lot and can never be merged into a single script.

**Comment by penteseter123** [comment]
Thanks Very Mach Problem Solved

**Pull request closed without merging**